### PR TITLE
Js text notebooks

### PR DESCRIPTION
**Pull request by khetan2**
*Issue #, if available:* JumpStart Text Classification Notebook

*Description of changes:*

*Testing done:*

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `tox -e black-format,black-nb-format`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,713 @@
+{
```
Comment (atqy) [inline introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb:1]: <p>@julkroll do we need have these files in sagemaker-sample-files to unify our data sources?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,687 @@
...
+    "# Delete the SageMaker endpoint and the attached resources\n",
+    "finetuned_predictor.delete_model()\n",
+    "finetuned_predictor.delete_endpoint()"
+   ]
+  }
+ ],
+ "metadata": {
+  "instance_type": "ml.t3.medium",
+  "kernelspec": {
+   "display_name": "Python 3 (Data Science)",
```
Comment (JGuinegagne) [inline introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:668]: would this work outside of Studio?

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,687 @@
...
+    "# Create the SageMaker model instance. Note that we need to pass Predictor class when we deploy model through Model class,\n",
+    "# for being able to run inference through the sagemaker API.\n",
+    "model = Model(\n",
+    "    image_uri=deploy_image_uri,\n",
+    "    source_dir=deploy_source_uri,\n",
+    "    model_data=base_model_uri,\n",
+    "    entry_point=\"inference.py\",\n",
+    "    role=aws_role,\n",
+    "    predictor_cls=Predictor,\n",
+    "    name=endpoint_name,\n",
```
Comment (JGuinegagne) [inline introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:230]: consider deploying in network isolation (see other comment below)

**Review by atqy — APPROVED** [review]

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,687 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "32a92ad2",
+   "metadata": {},
+   "source": [
+    "### 4.1. Retrieve JumpStart Training artifacts\n",
+    "***\n",
+    "Here, for the selected model, we retrieve the training docker container, the training algorithm source, the pre-trained model, and a python dictionary of the training hyper-parameters that the algorithm accepts with their default values. Note that the model_version=\"*\" fetches the lates model. Also, we do need to specify the training_instance_type to fetch train_image_uri.\n",
```
[the remark this answers is not in the document]
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:382]: Non-blocking. We will do in the next revision.

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,687 @@
...
+    "finetuned_predictor.delete_endpoint()"
+   ]
+  }
+ ],
+ "metadata": {
+  "instance_type": "ml.t3.medium",
+  "kernelspec": {
+   "display_name": "Python 3 (Data Science)",
+   "language": "python",
+   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:us-west-2:236514542706:image/datascience-1.0"
```
[the remark this answers is not in the document]
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:670]: Non-blocking. We will do in the next revision.

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb (hunk 2 above)
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:668]: Non-blocking. We will do in the next revision.

File: introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/README.md [hunk 6]
```diff
@@ -0,0 +1,2 @@
+### SageMaker JumpStart Sentence Pair Classification Training & Deployment
+This notebook `Amazon_JumpStart_Sentence_Pair_Classification.ipynb` for  Sentence Pair Classification task demonstrates (a) how to use a pre-trained model available in JumpStart for inference, and (b) how to fin-etune a pre-trained Transformer model on a custom dataset using JumpStart transfer learning algorithm, and how to use the fine-tuned model for inference.
```
Comment (jkroll-aws) [inline introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/README.md:2]: Typo: `fin-etune` -> `fine-tune`, extra space in `for  Sentence`

File: introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb [hunk 7]
```diff
@@ -0,0 +1,930 @@
...
```
Comment (jkroll-aws) [inline introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb:1]: <p>Please clear all notebook outputs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (jkroll-aws) [inline introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb:1]: <p>Typo: <code>lates</code> -&gt; <code>latest</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

File: introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/README.md (hunk 6 above)
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/README.md]: fixed.

File: introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb (hunk 7 above)
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb:1]: fixed.

Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_sentence_pair_classification/Amazon_JumpStart_Sentence_Pair_Classification.ipynb:1]: fixed.

File: introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb [hunk 8]
```diff
@@ -0,0 +1,687 @@
+{
```
[the remark this answers is not in the document]
Comment (khetan2) [reply introduction_to_amazon_algorithms/jumpstart_text_classification/Amazon_JumpStart_Text_Classification.ipynb:1]: fixed.

**Review by atqy — APPROVED** [review]

**Review by jkroll-aws — APPROVED** [review]

**Pull request merged**